store/src/java/com/zimbra/cs/util/AccountUtil.java

```diff
@@ -906,7 +906,8 @@ public static Account getRequestedAccount(String identityId, Account authAcct) t
         return null;
     }
 
-    public static boolean isZulipChatEnabled(Account account) {
+    // @TODO this is bad way of doing this, we could have used domainInherited flag on ldap attribute zimbraFeatureZulipChatEnabled
```

Comment:
Since we are not using this, we can remove commented code. And whenever we need it we can always get it from history :)

Reply:
Sure removed code as well as ldap attributes, next minor version upgrade can reintroduce ldap attributes